src/mpyl/steps/deploy/k8s/resources/traefik.py

```diff
@@ -41,7 +43,8 @@ def _interpolate_names(host: str, name: str) -> str:
             {
                 "kind": "Rule",
                 "match": _interpolate_names(
-                    host.host.host.get_value(target), host.name
+                    host=host.host.host.get_value(target),
```

Comment:
`host=host.host.host` ?!
Maybe we could rename one of these to `traefikHost` at least?  This class for example https://github.com/Vandebron/mpyl/blob/876c21a5c4b8d7f04e1322d83b397c865325679b/src/mpyl/project.py#L321-L320

Reply:
done in [90e307f](https://github.com/Vandebron/mpyl/pull/200/commits/90e307fc80c5e733e36f7ba7dafc39ae5c5e9622)